### PR TITLE
feat: [CDS-34090]: helm manifests and helm manifest stores runtime view revamped

### PR DESCRIPTION
**Pull request by bhavya-sinha**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by bhavya-sinha** [comment]
retrigger jest

**Comment by bhavya-sinha** [comment]
retrigger jest

File: src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ManifestSource/HelmManifestSource/HelmS3ManifestSource/HelmS3ManifestSource.tsx [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+      className={cx(css.inputWidth, css.layoutVerticalSpacing)}
+    >
+      {isFieldRuntime(`${manifestPath}.spec.store.spec.connectorRef`, template) && (
+        <div data-name="connectorRefContainer" className={css.verticalSpacingInput}>
+          <FormMultiTypeConnectorField
+            disabled={isFieldDisabled(`${manifestPath}.spec.store.spec.connectorRef`)}
+            name={`${path}.${manifestPath}.spec.store.spec.connectorRef`}
+            selected={get(initialValues, `${manifestPath}.spec.store.spec.connectorRef`, '')}
+            label={getString('connector')}
+            placeholder={''}
```
Comment (devinder-harness) [inline src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ManifestSource/HelmManifestSource/HelmS3ManifestSource/HelmS3ManifestSource.tsx:114]: is `placeholder` a required `prop` ?

**Review by devinder-harness — APPROVED** [review]

File: src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ManifestSource/HelmManifestSource/HelmS3ManifestSource/HelmS3ManifestSource.tsx (hunk 1 above)
Comment (bhavya-sinha) [reply src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ManifestSource/HelmManifestSource/HelmS3ManifestSource/HelmS3ManifestSource.tsx:114]: No, I will update in cupcoming PR. The components are just copied from ManifestInputForm(old component)

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**